### ISSUE TITLE
How to remove Cortana

### ISSUE THREAD

**Issue by Legionisgod** [ntdevlabs/tiny11builder#38]
How to remove Cortana

You can also remove Cortana from Command Prompt (admin) using the “winget uninstall –id 9NFFX4SZZ23L” command.

**Comment by sergiohzph** [comment]
Yeah I was about to suggest that, you can see all Windows 11 packages by typing "winget list".

**Comment by TheKingFireS** [comment]
I found other way with same method.
`dism /image:c:\scratchdir /Remove-ProvisionedAppxPackage /PackageName:Microsoft.549981C3F5F10_3.2204.14815.0_neutral_~_8wekyb3d8bbwe`
Only difference is package with version name.

**Comment by jack020202** [comment]
Do you think that this method can work also in windows 10?

**Comment by Legionisgod** [comment]
yes i think so
